Dockerfile.mysql

```diff
@@ -0,0 +1,10 @@
+FROM mysql:8.2.0
+
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+ENV MYSQL_ROOT_PASSWORD=root_password
+
+VOLUME ["/var/lib/mysql"]
+
+EXPOSE 3306
```

Comment:
Add Eof(End-of-file), need to add a blank line at the end
![image](https://github.com/user-attachments/assets/ba6b7862-663b-4532-b367-e31f29d32ace)